3.3/.sti/bin/assemble

```diff
@@ -1,23 +1,66 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
+  # For SCL enablement
+  source .bashrc
```

Comment:
Not trying to make an SDK, just make it easier (at least for me) to understand, especially for the lack of tests and the time it takes to run the ones with have for sti scripts.
I'm totally open to whatever format we'll have prior to merging this. Going through the comments one by one. Will continue on it tomorrow.